api/config.py

```diff
@@ -151,6 +164,18 @@ class Configuration(CoreConfiguration):
             "key": HELP_URI,
             "label": _("Patron support custom integration URI")
         },
+        {
+            "key": LARGE_COLLECTION_LANGUAGES,
+            "label": _("The primary languages represented in this library's collection")
+        },
+        {
+            "key": SMALL_COLLECTION_LANGUAGES,
+            "label": _("Other major languages represented in this library's collection")
+        },        
+        {
+            "key": TINY_COLLECTION_LANGUAGES,
+            "label": _("Other languages in this library's collection")
+        },        
```

Comment:
I think "type": "list" doesn't work here because that type is also expected to have an exhaustive list of "options" defined. I don't want to put up all the ISO-639-2 languages in a big list. (Although I can if it won't make the page unusable.)